test/dleq.test.ts

```diff
@@ -4,11 +4,11 @@
 // at https://opensource.org/licenses/BSD-3-Clause
 
 import { DLEQParams, DLEQProof, DLEQProver, Elt, Oprf, Scalar } from '../src/index.js'
-
+import './setGroup.js'
 import { serdeClass } from './util.js'
 
 describe.each(Object.entries(Oprf.Group.ID))('DLEQ', (groupName, id) => {
```

Comment:
```suggestion
import { GroupConsNoble } from '../src/groupNoble.js'
import { GroupConsSjcl } from '../src/groupSjcl.js'

describe.each([
    ["noble",GroupConsNoble],
    ["sjcl",GroupConsSjcl]
])(`Group-%s`,(_gropName :string, g :GroupCons)=>{
    Oprf.Group = g
describe.each(Object.entries(Oprf.Group.ID))('DLEQ', (groupName, id) => {
```

Reply:
I think maybe we could import the test declaration from some file that honours an env var.
Or perhaps jest has some inbuilt test filtering ? In any case, it's desirable to not have to sit through the slow sjcl tests